src/containers/EquipmentCheckoutPage/CheckoutPage.duck.js

```diff
@@ -161,6 +166,19 @@ export const stripeCustomerError = e => ({
 
 /* ================ Thunks ================ */
 
+export const checkTransaction = (transactionId) => (dispatch, getState, sdk) => {
```

Comment:
Concern about this coding style (directly return the data from thunk). Usually, we fetch data from sdk, then we store them to redux, and load redux state to components for using.